site/content/en/docs/concepts/fair_sharing.md

```diff
@@ -7,8 +7,70 @@ description: >
 ---
 
 
-### [Admission Fair Sharing](/docs/concepts/admission_fair_sharing)
+## [Admission Fair Sharing](/docs/concepts/admission_fair_sharing)
 
-A mechanism for ordering workloads based on the historical resource usage of their source LocalQueues, giving preference to those that have consumed fewer resources over time.
+A mechanism for ordering workloads based on the historical resource usage of
+their source LocalQueues, giving preference to those that have consumed fewer
+resources over time.
 
-### [Preemption based Fair Sharing](/docs/concepts/preemption/#fair-sharing)
\ No newline at end of file
+## [Preemption based Fair Sharing](/docs/concepts/preemption/#fair-sharing)
+
+### Proof that two workloads won't preempt each other
```

Comment:
nit: I don't feel too strongly about this but it could be useful to include the potential difference in preemption rates for the 2 different LessThanOrEqualToFinalShare / LessThanInitialShare stated here-> https://github.com/kubernetes-sigs/kueue/issues/6774#issuecomment-3299187094

Approving & will defer to whatever y'all think makes sense for docs

Reply:
WDYT @gabesaba , does it impact the proof? If so we can follow up.